src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1066,7 +1066,7 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			block.isNotColidable = _true;
 			block.canProvidePower = _true;
 			block.isBeaconBase = _true;
-			block.isWaterloggable = !_true;
+			block.isWaterloggable = false;

```

Comment:
Do not hardcode test conditions.

Reply:
Ok will revert to "!_true"

Reply:
It was more like a test, if it's the only problem or will it fail again. Also reverted.

Reply:
I will try to check why this is happening in the upcoming days.